.github/workflows/codespell_and_flake.yml

```diff
@@ -16,19 +16,30 @@ jobs:
       CODESPELL_SKIPS: 'doc/auto_*,*.fif,*.eve,*.gz,*.tgz,*.zip,*.mat,*.stc,*.label,*.w,*.bz2,*.annot,*.sulc,*.log,*.local-copy,*.orig_avg,*.inflated_avg,*.gii,*.pyc,*.doctree,*.pickle,*.inv,*.png,*.edf,*.touch,*.thickness,*.nofix,*.volume,*.defect_borders,*.mgh,lh.*,rh.*,COR-*,FreeSurferColorLUT.txt,*.examples,.xdebug_mris_calc,bad.segments,BadChannels,*.hist,empty_file,*.orig,*.js,*.map,*.ipynb,searchindex.dat,install_mne_c.rst,plot_*.rst,*.rst.txt,c_EULA.rst*,*.html,gdf_encodes.txt,*.svg'
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: GuillaumeFavelier/ci-skip@enh/separator
```

Comment:
We also should wait for this to get pushed upstream before merging

Reply:
Looks like it works, now we just wait for this before merge right?

Reply:
It works indeed. I'll try to fix my `docker` issues on https://github.com/mstachniuk/ci-skip/pull/4 then

Reply:
https://github.com/mstachniuk/ci-skip/pull/4 has been merged! Also the syntax changed slightly, instead of `separator`, it's `commit-filter-separator`. I updated the PR in a5288cf